src/rest.js

```diff
@@ -85,6 +89,13 @@ export default class RestClient extends Vue {
     return this.user;
   }
 
+  async fetchOauth(redirect = window.location.href, list = true) {
```

Comment:
One design decision we (@jbeezley @danlamanna and I) made regarding the `RestClient` class is that we don't want to populate it with wrapper methods for "ordinary" REST requests. Things like login/logout/register/fetchUser are provided because they actually change the state of the `RestClient` instance itself and emit special events. I could also see exceptions made for certain kinds of particularly nontrivial requests, though to be honest I have not really encountered any other than uploading files. Since this call is trivial, let's remove this method and just make the call directly from the component.